package.json

```diff
@@ -13,12 +13,13 @@
     "build:test": "yarn run clean && yarn run prep && NODE_ENV=test node --max_old_space_size=4096 node_modules/.bin/webpack --progress --colors",
     "build:staging": "yarn run clean && yarn run prep && yarn run compile:sass && NODE_ENV=staging node --max_old_space_size=4096 node_modules/.bin/webpack --progress --colors",
     "clean": "rm -rf build/",
-    "chimp-test": "yarn run chimp:setup & chimp chimp.config.js && yarn run chimp:teardown",
-    "chimp-circleci-test": "yarn run chimp:setup & chimp .circleci/chimp.circleci.config.js && yarn run chimp:teardown",
-    "chimp-test:watch": "yarn run chimp:setup & chimp chimp.config.js --watch && yarn run chimp:teardown",
```

Comment:
Can we delete these seemingly now unused scripts? `chimp:setup` and `chimp:teardown`

Reply:
Done